src/js/health-records/containers/UpdatePage.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+import { checkRefreshStatus } from '../actions/refresh';
+import { submitForm } from '../actions/form';
+
+import ProgressBar from '../../common/components/ProgressBar';
+
+export class UpdatePage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.pollRefresh = () => {
+      setInterval(() => {
+        props.checkRefreshStatus();
+      }, 10000);
+    };
+    this.handleSkipToDownload = this.handleSkipToDownload.bind(this);
+  }
+
+  componentDidMount() {
+    this.props.checkRefreshStatus();
+    this.pollRefresh();
+  }
+
+  componentWillReceiveProps(nextProps) {
+    const erroredUpdates = nextProps.refresh.statuses.ERROR;
+    if (erroredUpdates.length === 0) {
```

Comment:
Yes, this is a good point. I will follow up with @saneshark on what the success state looks like. Presumably the status should be OK, AND all of the dates should change to the date of the request. Right now this only checks for the statuses.